hyp3_metadata/templates/insar/readme.md.txt.j2

```diff
@@ -0,0 +1,298 @@
+
+*These files are optional. Select the "Include Look Vectors" option in the On Demand HyP3 Processing Options to include them in the product package.*
+
+----------------
+## 9. Incidence Angle Map
+
+The local incidence angle is defined as the angle between the incident radar signal and the local surface normal. The output is a GeoTIFF giving the local incidence angle in radians for each pixel in the scene.
+
+*This file is optional. Select the "Include Inc. Angle Map" option in the On Demand HyP3 Processing Options to include it in the product package.*
+
+----------------
+## 10. DEM used to process the data
+
+This is a copy of the Digital Elevation Model (DEM) layer used for processing. This layer is tagged with _dem.tif, and is a 32-bit float GeoTIFF.
+
+The source DEM has a geoid correction applied before it is used for processing, so height values in this file will differ from the source DEM. The DEM is down-sampled to twice the pixel spacing of the desired InSAR output pixel spacing before processing. As such, an 80-m InSAR product requires the DEM to be down-sampled to a pixel spacing of 160 m. It is provided in the same pixel spacing/alignment and projected to the same UTM Zone as the interferogram files.
+
+*This file is optional. Select the "Include DEM" option in the On Demand HyP3 Processing Options to include it in the product package.*
+
+The DEM used for this product is {{ dem_name }}, which has a native pixel spacing of {{ dem_pixel_spacing(dem_name) }}. {{ dem_blurb(dem_name) }}
+
+{% if dem_blurb(dem_name) %}
+*Refer to the _dem.tif.xml file for additional information about the specific DEM included with this product, including use and citation requirements.*
+
+{% endif %}
+----------------
+## 11. Water Mask
+
+The water mask specifies, pixel-by-pixel, where large waterbodies exist in the corresponding InSAR product. Pixel values of 0 indicate areas that should be masked out, while areas with a pixel value of 1 are included in the InSAR processing. This mask is applied to the DEM prior to running InSAR jobs, effectively clipping out the water, as areas with DEM heights of 0 are considered invalid for InSAR processing. The water mask is stored as a byte-valued GeoTIFF file and is included in the product package when water masking is applied.
+
+The water mask is not precise; land is buffered to reduce the possibility of near-shore features being excluded while reducing the impact of phase-unwrapping errors over large expanses of water. Only oceans, seas, and very large inland waterbodies are masked.
+
+*This file/processing is optional. Select the "Include Water Mask" option in the On Demand HyP3 Processing Options to apply water masking and include a copy of the mask in the product package.*
```

Comment:
I slightly prefer "Apply Water Mask" to "Include Water Mask"

Reply:
This will probably depend on how the water masking is implemented. There might end up being separate options for including and applying a water mask. I expect this part will be adjusted as decisions are made on the actual water masking implementation. I just went by what was currently listed in the readme for now.